metrics/code_size.py

```diff
@@ -0,0 +1,43 @@
+from byoqm.metric.metric import Metric
+from byoqm.metric.result import Result
+from byoqm.metric.violation import Violation
+from byoqm.source_repository.source_repository import SourceRepository
+from byoqm.source_repository.query_translations import translate_to
+
+
+class CodeSize(Metric):
```

Comment:
What is the reason behind changing the name of file length?

Reply:
Its a new metric. It counts the total number of lines and spits that out rather than the number of files being over 250 lines. I use this to calculate the technical debt ratio.